api/game.py

```diff
@@ -232,6 +293,28 @@ def get_player_info(player_names):
             good_roles_in_game.append("Tristan")
             good_roles_in_game.append("Iseult")
 
+    # if we took no lovers and not the older sibling:
+    # we must replace one role with a new role
+    # if there are not enough roles to replace, we remove the older sibling and one other role,
+    # and add the lovers
+    if (
+        sum(gr in ["Tristan", "Iseult"] for gr in good_roles_in_game) == 0
+        and "Older Sibling" not in good_roles_in_game
```

Comment:
Could these two lines not be condensed by including "Older Sibling" in the check?